.github/workflows/self-nightly-past-ci-caller.yml

```diff
@@ -30,6 +30,7 @@ jobs:
     with:
       framework: pytorch
       version: "1.13"
+      sha: ${{ github.sha }}

```

Comment:
The workflow file `self-past.yml`, when used in a `workflow_call`, accept some inputs including `sha`:

https://github.com/huggingface/transformers/blob/0670ccbf3a6de67b6b4abf44ba798771523654d5/.github/workflows/self-past.yml#L10-L22

Previously, the Past CI is launched manually by @ydshieh, and the goal is to targe a fixed & specific commit for some runs (so we can compare the results before/after the fixes). At that time, it was launched only once during a few months.

Right now, we run it twice per week, see #22393, it makes more sense to run the CI against latest `main`. Otherwise, the reports of the (automatically triggered) runs will contain failures that have been fixed on `main` and hard to understand at the 1st galance.

R